packages/mui-material-next/src/SnackbarContent/SnackbarContent.tsx

```diff
@@ -25,22 +25,20 @@ const SnackbarContentRoot = styled(Paper, {
   name: 'MuiSnackbarContent',
   slot: 'Root',
   overridesResolver: (props, styles) => styles.root,
-})(({ theme }) => {
-  const emphasis = theme.palette.mode === 'light' ? 0.8 : 0.98;
-  const backgroundColor = emphasize(theme.palette.background.default, emphasis);
+})<{ ownerState: SnackbarContentOwnerState }>(({ theme }) => {
+  const { vars: tokens } = theme;
 
   return {
     ...theme.typography.body2,
```

Comment:
@Best-Sardar We can break up this spread to use `theme.vars` as well, you can use https://github.com/mui/material-ui/blob/master/packages/mui-material-next/src/FormLabel/FormLabel.tsx#L49-L53 as a reference